doc/nrf/libraries/networking/wifi_mgmt_ext.rst

```diff
@@ -0,0 +1,30 @@
+.. _wifi_mgmt_ext:
+
+WiFi Management Extension
+#########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The WiFi Management Extension adds an automatic connection feature to the WiFi stack.
+
+Overview
+********
+
+The automatic connection feature is implemented by adding a `NET_REQUEST_WIFI_CONNECT_STORED` command to Zephyr's WiFi L2 Layer.
+Credentials are pulled from :ref:`_lib_wifi_credentials`.
+
+Configuration
+*************
+
+To use the WiFi credentials library, enable the :kconfig:option:`CONFIG_WIFI_MGMT_EXT` Kconfig option.
+
+For development purposes, you can set a static WiFi credentials configuration with :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC_SSID`, :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC_PASSWORD`, and :kconfig:option:`CONFIG_WIFI_CREDENTIALS_STATIC_TYPE`.
+
+Limitations
+***********
+
```

Comment:
The library has the following limitations:

Reply:
done